### DISCUSSION TITLE
L'automatisation est-elle compatible avec les plugins de tacking WordPress ?

### DISCUSSION THREAD

**Discussion by ArnaudKeyen** [AmauriC/tarteaucitron.js#773] [Q&A]
Salut Amauri,

Est-ce que l'option du Mode automatique permet de bloquer les services de tracking chargés avec d'autres plugins WordPress ? 

Je pense principalement au plugin : Google Site Kit et Facebook Pixel
Qui permettent de faciliter drastiquement la mise en place des codes de tracking.

Je te remercie pour ton retour.

**Comment by AmauriC** [comment]
Salut : ) 

Yes, le mode auto détecter et bloque automatiquement les requetes vers les services supporté par tarteaucitron.

Google analytics et Facebook en font partie.
